tests/test_config.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
In general, it is very good idea to add tests!  
But if you do so, then add a new section to <code>README.md</code>